### DISCUSSION TITLE
Bulk Changing Filtered Recipes?

### DISCUSSION THREAD

**Discussion by jonathan-morton** [mealie-recipes/mealie#4048] [Q&A]
Is it possible to bulk edit recipes based on a filter?

I am just getting started, and have imported my recipes from Paprika. All of the old Paprika organizers were added as tags. However I have created broad categories now and would like to remove some of the old tags. For example I now have a main category and would like to add everything with the main tag to the main category. I don't really see how to do this kind of bulk editing in the data management section, but maybe I'm missing something obvious.

**Comment by jjauzion** [comment]
I think there is already a feature request for this: #3652 that you can thumbs up ;)
